lib/aptible/cli/agent.rb

```diff
@@ -145,26 +145,51 @@ def login
 
           # If the user has added a security key and their computer supports it,
           # allow them to use it
-          if u2f && !which('u2f-host').nil?
+          # https://developers.yubico.com/libfido2/Manuals
+          if u2f && !which('fido2-assert').nil? && !which('fido2-token').nil?
```

Comment:
Where do users get these binaries for various platforms?

Reply:
https://github.com/Yubico/libfido2#installation